app/models/agents/weather_agent.rb

```diff
@@ -111,14 +90,17 @@ def coordinates
     end
 
     def language
-      interpolated['language'].presence || 'EN'
```

Comment:
Looks like there are 4 instead of 2 spaces used for the indentation.

Reply:
Will fix

Reply:
Fixed